weave-js/src/components/PagePanelComponents/Home/Browse3/datasets/SchemaMappingStep.tsx

```diff
@@ -0,0 +1,352 @@
+import {Box, Paper, Stack, Tooltip, Typography} from '@mui/material';
+import React, {useEffect, useState} from 'react';
+
```

Comment:
![Screenshot 2025-02-04 at 18 14 45](https://github.com/user-attachments/assets/6905a138-b552-4fc2-bd34-e0dc976091c6)
How is the grouping determined? Do you always group the same op, regardless of version? What if the schema changed between versions?

Reply:
We take the union of selected op version schema for the dropdown options. The selected calls are grouped by op version. So in the screen shot it looks like you selected two calls that use different versions of the same op.

Reply:
oh got it - makes sense!